### DISCUSSION TITLE
Q4 Summary blog post (10-01-2022)

### DISCUSSION THREAD

**Discussion by MabezDev** [MabezDev/blog#7] [Ideas]
Topics already planned below. Items marked with (?) may need more info before I can write about them.

- compiler
  - fixes & improvements
  - build updates (?) 
- probe-rs:
  - usb serial jtag support
  - new release includes our fixes
- atomic emulation trap handler
- STD builds
  - progress on native builds (?)
  - S3 support
- book updates
- espflash updates (?) 
- bare metal updates

**Comment by georgik** [comment]
- explain why is EP8266 not supported
- mention limitation on Windows regarding build path and suggested workaround with WSL2
- espflash mention problem with ESP32/JTAG - request community help?
- supported archs with Xtensa 1.57.0.2 - x86_64: Windows, Linux, macOS; - aarch64 macOS with LLVM 13 - next release scheduled to 25th Jan. 2022 https://github.com/esp-rs/rust-build/milestones
- ask readers to send links to examples of Rust apps on ESP32 - we can collect and publish links and include them in regression testing when releasing new version of Rust
